### DISCUSSION TITLE
unifi script error

### DISCUSSION THREAD

**Discussion by jovanradin** [tteck/Proxmox#2389] [Help Needed] [answered]
when i tried unifi script i got error, log from console:
Creating a Unifi LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-zfs for Container Storage.
 ✓ Updated LXC Template List
 ✓ Downloaded LXC Template
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 10.32.0.248 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 - Installing OpenJDK  /
[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Few weeks ago i installed unifi controller without problems. I can't find verbose mode in script (if $VERBOSE=yes)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/commit/09846607f21a7bbc94b6a7a5162419e06550ceb7 Clear cache
You enable verbose mode in Advanced settings

**Reply by jovanradin** [reply, marked as answer]
Solved, problem was with typo in sources.list, after fixing sorces.list script has finished succesfully.

Thanks :)

**Reply by tteck** [reply]
what sources list?

**Reply by jovanradin** [reply]
On proxmox node itself (/etc/apt/sources.list). 
I found error because proxmox reported that it didn't finish apt-get update.

**Reply by Rawet** [reply]
Can you specify what was wrong?
im having the exact same issue but i cant find anything wrong with my /etc/apt/sources.list

**Reply by nwebr** [reply]
Hi!
What i've to do? i've the same problem and i've no idea what i habe to do.
